### PR TITLE
Change last_tuned timestamp to be more precise

### PR DESCRIPTION
**Pull request by RobAtticus**
This timestamp will be stored in telemetry, so having the full
timestamp with timezone information helps us store it naturally in
the TimescaleDB database.

### CODE REVIEW

File: pkg/tstune/tuner.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ const (
 	fmtTunableParam = "%s = %s%s\n"
 	fmtLastTuned    = "timescaledb.last_tuned = '%s'"
 
-	lastTunedDateFmt = "2006-01-02 15:04"
+	lastTunedDateFmt = "2006-01-02 15:04:05.999999 -0700"
```
Comment (solugebefola) [inline pkg/tstune/tuner.go]: not familiar with the code...when running, will this be replaced with a dynamic value that is the last tuned time?

Comment (JLockerman) [reply pkg/tstune/tuner.go]: @solugebefola yup, go uses a fixed timestamp as a template to generate the actual output

Comment (solugebefola) [reply pkg/tstune/tuner.go]: or does this describe the format?

Comment (RobAtticus) [reply pkg/tstune/tuner.go]: Yeah, as @JLockerman said, Go uses a fixed string template to describe how a particular time should be printed. So this is `YYYY-MM-DD hh:mm:ss.uuuuuu timezone`, or ISO8601

Comment (cevian) [reply pkg/tstune/tuner.go]: Not sure it matters but I think postgres does not put a space between the nanos and the `-` for the timezone. Should we change just to be consistent?

**Comment by RobAtticus** [comment]
I can do it either, can you verify what PostgreSQL does? This format was taken from TSBS so I don't think PG has any problem correctly interpreting it

**Comment by svenklemm** [comment]
Why not use RFC3339 or RFC3339Nano as timeformat postgres can parse both

**Comment by RobAtticus** [comment]
Thanks @svenklemm, changed to that. Much simpler

**Pull request merged**